Nagstamon/QUI/__init__.py

```diff
@@ -3568,6 +3572,12 @@ def cell_clicked(self):
                 # action to signalmapper
                 self.signalmapper_action_menu.setMapping(action_menuentry, a)
                 action_menuentry.triggered.connect(self.signalmapper_action_menu.map)
+            SMHUB_USLESS_ACTION = ['Telnet','RDP','SSH','VNC'] 
```

Comment:
These actions can be deleted by user anyway - I think they should not be handled here specifically for SMHub.

Reply:
I have removed it in the last commit